operator/ARCHITECTURE.md

```diff
@@ -93,24 +91,26 @@ namespace is defined as:
 defaultNamespace: istio-system
 ```
 
-and namespaces are specialized for the security feature and one of the components:
+and namespaces are specialized for the gateway feature and its components:
 
 ```yaml
-security:
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-gateways
+spec:
   components:
-    namespace: istio-security
-    citadel:
-policy:
-  components:
-    policy:
+    ingressGateways:
+    - name: istio-ingressgateway
```

Comment:
Nit: is this example to deploy gateway to `istio-gateways` ns? If yes, we can add namespace here `namespace: istio-gateways`

Reply:
@morvencao yes, it is to show how to deploy to a namespace.   sorry I don't understand your comment... I had the namespace in line 100.

Reply:
I mean this example will deploy gateway into `istio-system` namespace, but your intention is to deploy gateway to `istio-gateways` namespace, right?

If yes, then I suggest to add line `namespace: istio-gateways` after line 104.

Reply:
sorry I missed your comment earlier!  It is addressed now. :)  PTAL